front/components/assistant/conversation/InputBar.tsx

```diff
@@ -608,6 +635,13 @@ export function AssistantInputBar({
                           agentListRef.current?.reset();
                           inputNode.setAttribute("ignore", "backspace");
                           inputNode.blur();
+                          setEmpty(
```

Comment:
Note the innputNode is a content editable inside a non contentEditable that is itself contentEditable (the colored mention while auto-completing).

The setEmpty can be called from here but that feels a bit weird?

Reply:
Do you actually need it? isn't that covered by the master `input` event above? Maybe you do.